### PR TITLE
Disable kill and tp commands without a distance limit

### PR DESCRIPTION
**Pull request by MoWobbler**

### CODE REVIEW

File: src/main/java/net/simpvp/Misc/DisableCmd.java [hunk 1]
```diff
@@ -88,13 +88,28 @@ public void onServerCommandEvent(ServerCommandEvent event) {
 		String cmd = split[0].trim().toLowerCase();
 		Cmd c = cmds.get(cmd);
 
+		boolean allowCommand = true;
+		if (cmd.equals("kill") || cmd.equals("tp")) {
+			for (String args: split) {
+				if (args.startsWith("@") && !(args.contains("distance=..") || args.contains("distance=0.."))) {
```
Comment (C4K3) [inline src/main/java/net/simpvp/Misc/DisableCmd.java]: I'd just check for `distance=`.

File: src/main/java/net/simpvp/Misc/DisableCmd.java [hunk 2]
```diff
@@ -88,13 +88,28 @@ public void onServerCommandEvent(ServerCommandEvent event) {
...
+					allowCommand = false;
+					break;
+				}
+			}
+		}
+
+		if (!allowCommand) {
+			Misc.instance.getLogger().info("Blocking command without a distance limit: " + cmd);
```
Comment (C4K3) [inline src/main/java/net/simpvp/Misc/DisableCmd.java:103]: Send a message to the sender as well.

File: src/main/java/net/simpvp/Misc/DisableCmd.java (hunk 1 above)
Comment (C4K3) [inline src/main/java/net/simpvp/Misc/DisableCmd.java]: There are other parameters such as `dx` and `x`. I think they should work as well?

**Comment by MoWobbler** [comment]
Hmm, I think it still might be fine to be required to have a distance limit. I feel like it could be ugly to check for x y z and dx dy dz. I'm having trouble thinking of a good way to do that. Also I added the execute command

**Pull request closed without merging**